xls/dslx/stdlib/BUILD

```diff
@@ -59,6 +59,11 @@ xls_dslx_prove_quickcheck_test(
     srcs = ["std.x"],
 )
 
+xls_dslx_library(
```

Comment:
can we get a separate PR for that?

Reply:
Moved to a separate PR: https://github.com/google/xls/pull/2033

This PR will be rebased on top of #2033